src/components/assets/EvmAssetList.vue

```diff
@@ -1,56 +1,77 @@
 <template>
-  <div v-if="isShibuya || tokens" class="container">
-    <div class="row">
-      <span class="text--title">{{ $t('assets.assets') }}</span>
-    </div>
+  <div>
+    <div v-if="isShibuya || tokens" class="container">
```

Comment:
I didn't see this before, but now I look at it, we have `isShibuya`, `tokenSymbol === 'SDN' ? 'icons/sdn-token.png' : 'icons/astar.png'`, and `tokenSymbol === 'SBY' ? 'Shibuya' : currentNetwork`. This looks like a very complicated unending if-else anti-pattern. Please clean this if possible (if it's difficult for now, please open a ticket about this so we can improve it later)

Reply:
Fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/291/commits/29b1d6adf60164b8e91bbdbd7c248891ff1a4c48) commit.